week-2/interpret/invert.js

```diff
@@ -15,13 +15,18 @@ function invert(obj) {
 
   return invertedObj;
 }
-
+console.log(invert({ a: 1}));
 // a) What is the current return value when invert is called with { a : 1 }
+// Ans. {a: 1}
 
 // b) What is the current return value when invert is called with { a: 1, b: 2 }
+// Ans. (a: 1, b: 2)
 
 // c) What is the target return value when invert is called with {a : 1, b: 2}
+// Ans. { "1": "a", "2": "b"}
 
 // c) What does Object.entries return? Why is it needed in this program?
+// Ans. This is accessing the array and it is required to go through each element of array 
```

Comment:
This answer could be slightly clearer about what the array is relative to the object